text/0062-recoverable-scis.md

```diff
@@ -0,0 +1,112 @@
+- Feature Name: recoverable_scis
+- Start Date: 2023-03-02
+- MCIP PR: [mobilecoinfoundation/mcips#0062](https://github.com/mobilecoinfoundation/mcips/pull/0062)
+- Tracking Issue: (link to tracking issue)
+
+# Summary
+
+[summary]: #summary
+
+This MCIP proposes that inputs to be used in an SCI first be sent to a reserved subaddress `SCI_SUBADDRESS_INDEX`, extending the scheme from [MCIP #36](0036-reserved-subaddresses.md)
+
+This MCIP also proposes that new RTH memos be created to further support inputs used in SCIs, extending the scheme from [MCIP #4](0004-recoverable-transaction-history.md)
+
+# Motivation
+
+[motivation]: #motivation
+
+Similar to the motivations in [MCIP #32](0032-fog-compatible-gift-codes.md), a concern for users is the ability to track things across linked devices using the blockchain (and not separate services). It is a concern wrt txos used as an SCI in that one instance of an account could use a txo as an SCI and before the recipient of said SCI completes a transaction with it (which could take some arbitrary yet non trivial amount of time), another (or even the same, if data is not properly tracked) instance of the account could use that txo as an input in another transaction, therefore unintentionally rendering the SCI cancelled.
+
+Another concern is that because there are multiple types of ways to specify the rules of an SCI (one example being [MCIP #42](0042-partial-fill-rules.md) which extended the SCI rules to include partial fills) one might want to have some record of that information on the blockchain (instead of relying on a secondary service)
+
+# Guide-level explanation - WIP (awaiting feedback and discussion)
+
+[guide-level-explanation]: #guide-level-explanation
+
+When one wishes to generate an SCI, the app must first make a self payment to the `SCI_SUBADDRESS_INDEX = u64::MAX - 3`, which is now reserved for `TxOut`s used as `SCI`s.
+Apps should not select `TxOut`s at this subaddress for inputs in other transactions, and they may choose not to include these in the account balances as they represent the amount of active `SCI`s.
+
+The `TxOut` generated for use as the SCI must contain an `SCI Generation Memo` to identify the details of the SCI it was included in (specifics TBD as feature evolves from feedback and discussion)
+
+After an SCI is sent, an app can check if it has been claimed based on if this `TxOut` has been spent. Until it has been spent, the app can potentially cancel the `SCI` by sending a self payment with the `TxOut`.
```

Comment:
I think it's worth elaborating on how the app differentiates between canceling an SCI and the SCI being used. In both cases the TxOut will get spent.